glean-core/pings.yaml

```
@@ -34,6 +34,11 @@ baseline:
       *Note*: this ping will not contain the `glean.baseline.duration` metric.
     background: |
       The ping was submitted before going to background.
+    foreground: |
+      The ping was submitted when the application went to foreground, which
+      includes when the application starts.
+
+      The `glean.baseline.duration` metric is never sent in these pings.
```

Should this just be the same as for `dirty_startup`? Makes it easier to see the similarity.

```suggestion
      *Note*: this ping will not contain the `glean.baseline.duration` metric.
```